compiler/testData/diagnostics/tests/callableReference/kt32267.kt

```diff
@@ -1,4 +1,3 @@
-// FIR_IDENTICAL
```

Comment:
At least this change looks very suspicious

Reply:
Not really, we just expose hidden, unresolved references. That is all about https://youtrack.jetbrains.com/issue/KT-43948. See my other comments in the newly added .fir.kr (slightly right upper side).

Also, if you see the 1st commit separately, there are more cases that we drop `// FIR_IDENTICAL`. Luckily, all the others were recovered by the 2nd commit.

Reply:
Probably you're right, yes.